app/views/layouts/ucsdlib.html.erb

```diff
@@ -5,8 +5,9 @@
 		<%= render :partial => 'layouts/twitter_card' %>
 		<link href="<%= opensearch_catalog_path(:format => 'xml', :only_url => false) %>" title="<%= application_name%>" type="application/opensearchdescription+xml" rel="search"/>
 
-	  <script src="//libraries.ucsd.edu/jwplayer/jwplayer.js"></script>
-	  <script src="//libraries.ucsd.edu/jwplayer/key.js"></script>
+        <script src="//libraries.ucsd.edu/jwplayer/jwplayer.js"></script>
+        <script src="//libraries.ucsd.edu/jwplayer/key.js"></script>
+        <script src="https://www.mapquestapi.com/sdk/leaflet/v2.2/mq-map.js?key=zrnNkNZDkELMvViepTqFjZP8bEKxjDaG"></script>

```

Comment:
that's a very good point. the main benefit i can see of having it as an ENV variable (but perhaps not in Ansible vault) is if we ever need to change keys we might expect to look there first. But, whatever you think.